apache/zeppelin
# [ZEPPELIN-2702] save notes in reader-friendly format with zpln extension for VFS and Git repo

### What is this PR for?
first task in converting note files into human readable and Zeppelin own format

### What type of PR is it?
Improvement | Feature

### Todos
* [x] - initial flow
* [x] - rename through save
* [x] - tests for vfs

### What is the Jira issue?
[ZEPPELIN-2702](https://issues.apache.org/jira/browse/ZEPPELIN-2702)

### How should this be tested?
1. go create new notes or rename title of old ones
2. see the notebook directory, under [NOTE_ID]/title.zpln

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? separately
